ArtOfIllusion/src/artofillusion/LayoutWindow.java

```diff
@@ -811,6 +813,18 @@ public void showPopupMenu(Widget w, int x, int y)
     popupMenu.show(w, x, y);
   }
 
+
+  @Override
+  public void setVisible(boolean visible)
+  {
+    System.out.println("svc....");
+    if (visible && !hasNotifiedPlugins)
+    {
+      hasNotifiedPlugins = true;
+      PluginRegistry.notifyPlugins(Plugin.class, Plugin.SCENE_WINDOW_CREATED, this);
```

Comment:
I think these are reversed. We probably shouldn't claim that we've notified plugins until we've... notified plugins.